### PR TITLE
Graph visualizations

### PR DESCRIPTION
**Pull request by komalsingh0925**
1. Follow https://github.com/trevorwinser/Infinite-Loopers/blob/admin-dashboard-backend/Instructions/flask.md

2. Go to http://127.0.0.1:5000/data-visualization

3. Go to http://127.0.0.1:5000/data-visualization/94/8

4. While Flask is running, open visualization_test.py. There should be a green check mark shown at the bottom of the code. Click that and the tests will automatically run.

_P.S Don't worry if filter_visualization.html or data_visualization.html highlight red. They don't cause any issues with the functionality of the system._

image

#129

### CODE REVIEW

**Review by noahstaz — CHANGES_REQUESTED** [review]
Merge conflicts should be looked at, __pycache should be added to gitignore, and then the app.py ones would be a more specific change

**Comment by julieflament** [comment]
personally struggling with tests, but once again i think that is just an issue with my computer. from what i can see they look good. visualizations are great!

**Comment by julieflament** [comment]
resolved conflicts but make sure to check all the files before merging.

File: code/test/hurriscan.db
Comment (noahstaz) [inline code/test/hurriscan.db:1]: This should not be here I think your gitignore is outdated, update to the one on main before merge

**Review by noahstaz — APPROVED** [review]
Approved to merge if you update gitignore everything else is good and tests worked properly

**Pull request merged**
